docs/examples/pandapower_example.ipynb

```diff
@@ -84,7 +84,16 @@
    "metadata": {
     "scrolled": true
    },
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "c:\\Users\\AL29065\\AppData\\Local\\miniconda3\\envs\\power-grid-io-env\\lib\\site-packages\\pandapower\\create.py:4815: FutureWarning: Setting an item of incompatible dtype is deprecated and will raise in a future error of pandas. Value 'Dyn' has dtype incompatible with float64, please explicitly cast to a compatible dtype first.\n",
+      "  net[element_type].at[index, column] = value\n"
```

Comment:
please create a new issue for this. either we need to fix it if it is in PGM-IO, else if it is in pandapower, we should suppress this warning

Reply:
Discussed during stand-up